src/applications/vaos/tests/services/appointment/index.v2.unit.spec.js

```diff
@@ -831,11 +831,11 @@ describe('VAOS Appointment service', () => {
         [
           { op: 'remove', path: ['description'] },
           { op: 'remove', path: ['practitioners'] },
-          {
-            op: 'replace',
-            path: ['created'],
-            value: moment().format('YYYY-MM-DD'),
-          },
+          // {
+          //   op: 'replace',
+          //   path: ['created'],
+          //   value: moment().format('YYYY-MM-DD'),
+          // },
```

Comment:
I think you can remove these lines since v0 and v2 both have the same date type for the 'create` attribute

Reply:
Done